include/zephyr/drivers/sensor.h

```diff
@@ -201,7 +201,12 @@ enum sensor_channel {
 	SENSOR_CHAN_GRAVITY_VECTOR,
 	/** Gyroscope bias (X/Y/Z components in radians/s) */
 	SENSOR_CHAN_GBIAS_XYZ,
-
+	/** Heart rate value (bpm) */
+	SENSOR_CHAN_HEALTH_HEARTRATE,
```

Comment:
I'd drop the health since its a heartrate. Does not matter what your use case is really.